packages/core/test-helpers/core-test-helpers-model-versions/src/elasticsearch.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { createTestServers, type TestElasticsearchUtils } from '@kbn/core-test-helpers-kbn-server';
+
+/**
+ * Start the traditional ES cluster and return the instance.
+ */
+export const startElasticsearch = async ({
+  basePath,
+  dataArchive,
+  timeout,
+}: {
+  basePath?: string;
+  dataArchive?: string;
+  timeout?: number;
+} = {}): Promise<TestElasticsearchUtils> => {
+  const { startES } = createTestServers({
+    adjustTimeout: (t: number) => jest.setTimeout(t + (timeout ?? 0)),
+    settings: {
```

Comment:
I only implemented connecting to traditional ES. TBH I didn't really feel the need to have these tests running against serverless.

But we can add it later if we find a reason to do so.

Reply:
Out of an abundance of caution I would prioritise a serverless ES instance first.

If we were to "address" this would the intention be to add an option to the test bed factory?